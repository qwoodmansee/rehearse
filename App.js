import Navigator from '@navigation/src/components/navigator';
import React from 'react';

export default function App() {
  return (
    <Navigator />
  );
}
