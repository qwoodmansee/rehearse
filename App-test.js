import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

jest.mock('@navigation/src/components/navigator', () => 'Navigator');
     
describe('App', () => {  
  describe('using a mocked navigator', () => {
    it(`renders the mocked app`, () => {
      const tree = renderer.create(<App />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
