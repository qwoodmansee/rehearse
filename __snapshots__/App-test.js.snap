// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App using a mocked navigator renders the mocked app 1`] = `<Navigator />`;
